Lesson16/CallableFromTask1.java

```diff
@@ -0,0 +1,23 @@
+package Lesson16;
+
+import java.util.Random;
+import java.util.concurrent.Callable;
+
+public class CallableFromTask1 implements Callable<String> {
+    private String name;
+
+    public String getName() {
+        return name;
+    }
+
+    public CallableFromTask1(String name) {
+        this.name = name;
+    }
+
+    @Override
+    public String call() throws Exception{
+        Random generator = new Random();
+        Integer randomNumber = generator.nextInt(100);
+        return (this.getName() + ": " + randomNumber);
```

Comment:
коли є можливість використати примітиви то приорітет їм, ну і зайві дужки тільки додають складність для читання колу, але це мінорні штуки
```suggestion
        int randomNumber = generator.nextInt(100);
        return this.getName() + ": " + randomNumber;
```

Reply:
Оновлено, дякую за підказку